app/views/shared/post/_comment.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="comments">
+  <% post.comments.each do |comment| %>
+    <p><%= comment.author.name %> : <%= comment.text %></p>
+  <% end %>
+</div>
```

Comment:
- [ ] Currently you are iterating through all comments, but it should show only 5 recent comments, you already have a method called `last_five_comments` in Post model you can easily use it here:

`<% post.last_five_comments.each do |comment| %>`